src/test/java/uk/gov/pay/api/service/CardPaymentSearchServiceTest.java

```diff
@@ -146,7 +147,23 @@ public void searchShouldReturnAResponseWithOneTransaction_whenFilteringByCardHol
                 .assertThat("_links", hasKey("self"))
                 .assertThat("_links", hasKey("first_page"))
                 .assertThat("_links", hasKey("last_page"))
+                .assertThat("results[0]._links.capture", is(nullValue()))
                 .assertNotDefined("_links.next_page")
                 .assertNotDefined("_links.prev_page");
     }
+
+    @Test
+    @PactVerification({"connector"})
+    @Pacts(pacts = {"publicapi-connector-search-payment-with-charge-in-awaiting-capture-state"})
+    public void searchShouldReturnAResponseWithCaptureLinkPresent() {
+        Account account = new Account("123456", TokenPaymentType.CARD);
+        Response response =
+                paymentSearchService.doSearch(account, null, null,
+                        null, null, null,
+                        null, null, null,
+                        null, null, null, null);
+        JsonAssert.with(response.getEntity().toString())
+                .assertThat("results[0]._links", hasKey("capture"))
+                .assertThat("results[0]._links.capture.method", is("POST"));
+    }
```

Comment:
It would be nice to have a test to check we get {{"capture": null}} if the payment is not in the {{AWAITING CAPTURE REQUEST}} state